documentation/modules/metrics/proc_metrics-deploying-prometheus-operator.adoc

```diff
@@ -12,17 +12,20 @@ To deploy the Prometheus Operator to your Kafka cluster, apply the YAML bundle r
 
 . Download the `bundle.yaml` resources file from the repository.
 +
+[source,shell,subs="+quotes,attributes+"]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml > prometheus-operator-deployment.yaml
++
+** Update the namespace by replacing the example `my-namespace` with your own
++
 On Linux, use:
 +
 [source,shell,subs="+quotes,attributes+"]
-curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml | sed -e '/[[:space:]]\*namespace: [a-zA-Z0-9-]*$/s/namespace:[[:space:]]\*[a-zA-Z0-9-]*$/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
+sed -E -i '/[[:space:]]\*namespace: [a-zA-Z0-9-]*$/s/namespace:[[:space:]]\*[a-zA-Z0-9-]*$/namespace: _my-namespace_/' prometheus-operator-deployment.yaml
 +
 On MacOS, use:
 +
 [source,shell,subs="+quotes,attributes+"]
-curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml | sed -e '' '/[[:space:]]\*namespace: [a-zA-Z0-9-]*$/s/namespace:[[:space:]]\*[a-zA-Z0-9-]*$/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
-+
-** Replace the example `namespace` with your own.
+sed -i '' -e '/[[:space:]]\*namespace: [a-zA-Z0-9-]*$/s/namespace:[[:space:]]\*[a-zA-Z0-9-]*$/namespace: _my-namespace_/' prometheus-operator-deployment.yaml
```

Comment:
This does not seem to work for me. Are you sure this is using the MacOS `sed`?

Reply:
@scholzj , could you try the following command again? Thanks.

`sed -i '' -e '/[[:space:]]*namespace: [a-zA-Z0-9-]*$/s/namespace:[[:space:]]*[a-zA-Z0-9-]*$/namespace: my-namespace/' prometheus-operator-deployment.yaml`

Reply:
Yes, this command works for me.

Reply:
Great! And that's the command will display on the doc UI. The command showed on the adoc containing escaped back slash , ex: `\*`, which cannot be recognized by sed command. 

For your reference, this is what we saw on UI:
![image](https://github.com/strimzi/strimzi-kafka-operator/assets/43372967/04a4c9d2-8ffc-48b2-b506-6ee2a86690eb)

And this is the command in adoc file: `'/[[:space:]]\*namespace:` (i.e. no back slash)

Reply:
Ahh, sorry. My mistake. I did not realized it.